xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -221,14 +221,22 @@
         <Parameter Name="isFinalBlock" Type="System.Boolean" Index="4" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="charsConsumed">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="isFinalBlock">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">A source buffer containing the characters to encode.</param>
+        <param name="destination">The destination buffer to which the encoded form of <paramref name="source" /> will be written.</param>
+        <param name="charsConsumed">The number of characters consumed from the <paramref name="source" /> buffer.</param>
+        <param name="charsWritten">The number of characters written to the <paramref name="destination" /> buffer.</param>
+        <param name="isFinalBlock"><see langword="false" /> to indicate there is further source data that needs to be encoded; otherwise, <see langword="true" />.</param>
```

Comment:
```suggestion
        <param name="isFinalBlock"><see langword="true" /> to indicate the source buffer contains the entire data to encode; otherwise, <see langword="false" />.</param>
```

Reply:
Thanks Ahson. I think out of the two options, Ron's suggestion feels slightly more natural.